pallets/parachain-system/src/lib.rs

```diff
@@ -441,21 +434,24 @@ pub mod pallet {
 		Unauthorized,
 	}
 
-	/// We need to store the new validation function for the span between
-	/// setting it and applying it. If it has a
-	/// value, then [`PendingValidationCode`] must have a real value, and
-	/// together will coordinate the block number where the upgrade will happen.
-	#[pallet::storage]
-	pub(super) type PendingRelayChainBlockNumber<T: Config> =
-		StorageValue<_, RelayChainBlockNumber>;
```

Comment:
Just a note about this.

If we remove a storage value, we typically need to remove it from the storage. Otherwise, this value will become unreachable and will stay as garbage on-chain. However, if I am reading the code right, right after the previous version enacts the upgrade it will remove `PendingRelayChainBlockNumber`. Therefore, no clean up is needed.